components/sticky-footer.tsx

```
@@ -79,24 +94,41 @@ export function StickyFooter() {
-                  onMouseEnter={(e) => ((e.target as HTMLElement).style.color = "rgba(18, 17, 19, 0.8)")}
-                  onMouseLeave={(e) => ((e.target as HTMLElement).style.color = "#121113")}
+                  onMouseEnter={(e) =>
+                    ((e.target as HTMLElement).style.color =
+                      "rgba(18, 17, 19, 0.8)")
+                  }
+                  onMouseLeave={(e) =>
+                    ((e.target as HTMLElement).style.color = "#121113")
+                  }
                 >
-                  Github
+                  <a href="https://github.com/tokenist" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)